### PR TITLE
21 better structure of readme

### LINKED ISSUE #21: Better structure of README
Our documentation is getting bigger and bigger and therefore a single README does not seem to be the best fit anymore.
We need to find a solution for better (structured) documentation.

### PR DESCRIPTION
**Pull request by munichbughunter**
Fixes #21

### CODE REVIEW

File: docs/guides/assets/screenplay_pattern.png
Comment (csczygiol) [inline docs/guides/assets/screenplay_pattern.png:1]: During darkmode the connections arent visible

Comment (csczygiol) [inline docs/guides/assets/screenplay_pattern.png:1]: During darkmode the connections arent visible

File: docs/guides/basic_test_setup/actor_fixture.md
Comment (csczygiol) [inline docs/guides/basic_test_setup/actor_fixture.md:1]: Missing comments

File: docs/screenplay_pattern_overview/screenplay_pattern.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+We can also instruct Andy to use actions to navigate to the web interface of our system under test:
+
+```typescript
+await Andy.attemptsTo(Navigate.to('https://google.de'));
+```
+
+### Questions
+
+Apart from enabling interactions, abilities also enable actors to answer questions about the state of the system under test. 
+For example, we could instruct Ute to answer the question about the last response status.
```
Comment (kbrueckner) [inline docs/screenplay_pattern_overview/screenplay_pattern.md:84]: ... Ute to ask ...

File: docs/screenplay_pattern_overview/screenplay_pattern.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+```typescript
+await Ute.asks(Response.has.statusCode(response, 200));
+```
+
+Even though Andy uses a web browser and not an API client, the way we instruct him to answer web questions and perform web assertions is consistent with how you'd interact with any other interface:
+
+```typescript
+await Andy.asks(Element.toBe.visible(HomeScreen.LOGGED_IN_INDICATOR));
+```
+
```
Comment (kbrueckner) [inline docs/screenplay_pattern_overview/screenplay_pattern.md:95]: Maybe some chapter ending sentence

File: docs/guides/basic_test_setup/actor_fixture.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+[Back to overview](../guides.md)
+
+# Actor as a Fixture
+
+## Introduction
+
+Playwright Test is based on the concept of test fixtures. Test fixtures are used to establish environment for each test, giving the test everything it needs and nothing else. Test fixtures are isolated between tests. With fixtures, you can create actors based on their abilities.
```
[the remark this answers is not in the document]
Comment (kbrueckner) [reply docs/guides/basic_test_setup/actor_fixture.md:7]: With fixtures you can create actors which are then available in every test and give them abilities which they can use.

File: docs/guides/basic_test_setup/writing_tests.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+[Back to overview](../guides.md)
+
+# Writing tests
+
+Testla tests are simple, they
+    - perform tasks, and
```
Comment (kbrueckner) [inline docs/guides/basic_test_setup/writing_tests.md:6]: the - do not render as bullet points

File: docs/guides/advanced_test_setup/create_action.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+## Define custom Action
+
+Here, we define a simple reload action which can be performed by an actor.
+
+```typescript
+import { Actor, Action } from '@testla/screenplay-playwright';
+import { BrowseTheWeb } from '@testla/screenplay-playwright/web';
+
+export class Reload extends Action {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (kbrueckner) [inline docs/guides/advanced_test_setup/create_action.md:16]: remove that eslint comment

File: docs/guides/advanced_test_setup/create_action.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        const page = BrowseTheWeb.as(actor).getPage();
+        return page.reload();
+    }
+
+    public static page(): Reload {
+        return new Reload();
+    }
+}
+```
+
```
Comment (kbrueckner) [inline docs/guides/advanced_test_setup/create_action.md:27]: We need some explaination on what happens in the internal functions

File: docs/guides/advanced_test_setup/create_task.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+import { Navigate, Fill, Click, Wait } from '@testla/screenplay-playwright/web';
+import { LoginScreen } from '../screen/login-screen';
+
+export class Login extends Task {
+    private constructor() {
+        super();
+    }
+
+    public async performAs(actor: Actor): Promise<any> {
+        return await actor.attemptsTo(
```
Comment (kbrueckner) [inline docs/guides/advanced_test_setup/create_task.md:24]: the await should be removed

File: docs/guides/advanced_test_setup/create_task.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+            Wait.forLoadState('networkidle'),
+        );
+    }
+
+    public static toApp(): Login {
+            return new Login();
+    }
+}
+```
+
```
Comment (kbrueckner) [inline docs/guides/advanced_test_setup/create_task.md:38]: some explanations for the internal functions

File: docs/guides/advanced_test_setup/create_question.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+## Implementing custom questions
+
+You can also implement custom questions when needed. To do that, take a look on the following code.
+
+```typescript
+import { expect } from '@playwright/test';
+import { Question } from '@testla/screenplay-playwright';
+
+export class Entity extends Question<boolean> {
+    // eslint-disable-next-line
```
Comment (kbrueckner) [inline docs/guides/advanced_test_setup/create_question.md:17]: remove all eslint and ts comments

File: docs/guides/advanced_test_setup/create_question.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+
+    public async answeredBy(): Promise<boolean> {
+        const { obj, attribute, value } = this.payload;
+        const attributeFound = obj[attribute] === value;
+        
+        expect(attributeFound).toBe(this.checkMode === 'has');
+        return Promise.resolve(true);
+    }
+
+    /**
```
Comment (kbrueckner) [inline docs/guides/advanced_test_setup/create_question.md:36]: remove nonsense jsdoc comments

File: docs/guides/advanced_test_setup/create_question.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+        return new Entity('hasNot');
+    }
+
+    public attributeValue(obj: any, attribute: string, value: any): Entity {
+        this.payload = { obj, attribute, value };
+        return this;
+    }
+}
+```
+
```
Comment (kbrueckner) [inline docs/guides/advanced_test_setup/create_question.md:56]: function explanations

File: docs/guides/advanced_test_setup/create_action.md (hunk 5 above)
Comment (munichbughunter) [reply docs/guides/advanced_test_setup/create_action.md:16]: eslint comment removed.

File: docs/guides/basic_test_setup/actor_fixture.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (munichbughunter) [reply docs/guides/basic_test_setup/actor_fixture.md:7]: Added your suggestions.

File: docs/guides/basic_test_setup/screen.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
+[Back to overview](../guides.md)
+
+# Screen
+
+The `screen` represents elements/static properties that can be used to identify and interact with on a screen.
+It is designed to make the automation of actions more convenient.
+
+```typescript
+export class LoginScreen {
+    static EMAIL_INPUT = "email-input locator";
```
[the remark this answers is not in the document]
Comment (munichbughunter) [reply docs/guides/basic_test_setup/screen.md:10]: Done

File: docs/guides/basic_test_setup/screen.md
[the remark this answers is not in the document]
Comment (munichbughunter) [reply docs/guides/basic_test_setup/screen.md:1]: We don't show how the locator can look like. We wanna give an example how the Screen can look.

File: docs/guides/advanced_test_setup/create_action.md (hunk 6 above)
Comment (munichbughunter) [reply docs/guides/advanced_test_setup/create_action.md:26]: What do you think?

File: docs/guides/advanced_test_setup/create_task.md (hunk 8 above)
Comment (munichbughunter) [reply docs/guides/advanced_test_setup/create_task.md:38]: What do you think?

File: docs/guides/advanced_test_setup/create_question.md (hunk 11 above)
Comment (munichbughunter) [reply docs/guides/advanced_test_setup/create_question.md:48]: What do you think?

File: docs/guides/basic_test_setup/actor_fixture.md
Comment (munichbughunter) [reply docs/guides/basic_test_setup/actor_fixture.md:1]: Please check again.

**Review by kbrueckner — CHANGES_REQUESTED** [review]

File: docs/screenplay_elements/Web/Actions/clear.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+
+### Methods
+
+### performAs
+
+```typescript
+public async performAs(actor: Actor): Promise<any>;
+```
+
+- **Description:** Waits for either a specified loading state or for a selector to become visible/active and then clears all browser cookies using the `BrowseTheWeb` ability.
```
[the remark this answers is not in the document]
Comment (munichbughunter) [reply docs/screenplay_elements/Web/Actions/clear.md:26]: Fixed.

File: docs/screenplay_elements/Web/Actions/click.md [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+
+### on
+
+```typescript
+public static on(selector: Selector, options?: SelectorOptions): Click;
+```
+
+- **Description:** Creates a new instance of the `Click` class specifically for clicking on an element specified by a selector.
+- **Parameters:**
+  - `selector` - The string representing the selector.
```
[the remark this answers is not in the document]
Comment (munichbughunter) [reply docs/screenplay_elements/Web/Actions/click.md:39]: Selector.

File: docs/screenplay_elements/Web/Actions/get.md [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+
+### Methods
+
+### performAs
+
+```typescript
+public async performAs(actor: Actor): Promise<any>;
+```
+
+- **Description:** Waits for either a specified loading state or for a selector to become visible/active. Retrieves cookies, session storage items, or local storage items based on the specified mode.
```
[the remark this answers is not in the document]
Comment (munichbughunter) [reply docs/screenplay_elements/Web/Actions/get.md:28]: Nope. Fixed.

File: docs/screenplay_elements/Web/Actions/wait.md [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+  - [Class Overview](#class-overview)
+    - [Properties](#properties)
+    - [Methods](#methods)
+    - [performAs](#performas)
+    - [forLoadState](#forloadstate)
+    - [forSelector](#forselector)
+
+## Class Overview
+
+### Properties
```
[the remark this answers is not in the document]
Comment (munichbughunter) [reply docs/screenplay_elements/Web/Actions/wait.md:20]: removed

File: docs/screenplay_elements/Web/Miscellaneous/utils.md [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+
+### Functions
+
+### getSubLocator
+
+```typescript
+const getSubLocator = async (locator: Locator, subLocator?: Locator, text?: string | RegExp): Promise<Locator>;
+```
+
+- **Description:** Deals with selector == Playwright Locator and options.hasText.
```
[the remark this answers is not in the document]
Comment (munichbughunter) [reply docs/screenplay_elements/Web/Miscellaneous/utils.md:27]: Wie klingt das für dich?
```suggestion
- **Description:** The function filters for a locator based on the provided subLocator and text or regular expression.
```

**Review by kbrueckner — APPROVED** [review]

**Pull request merged**